openfisca_us/variables/finance/benefit/snap.py

```diff
@@ -151,10 +153,13 @@ class snap(Variable):
 
     def formula(spm_unit, period, parameters):
         # TODO: Add gross and net income checks.
-        return spm_unit("snap_max_benefit", period) - spm_unit(
+
+        res = spm_unit("snap_max_benefit", period) - spm_unit(
```

Comment:
```suggestion
        return spm_unit("snap_max_benefit", period) - spm_unit(
```
don't need the `res` here